app/code/Magento/Sitemap/Model/ResourceModel/Catalog/Product.php

```diff
@@ -396,9 +396,28 @@ protected function _getAllProductImages($product, $storeId)
      * Get media config
      *
      * @return \Magento\Catalog\Model\Product\Media\Config
+     * @deprecated No longer used, as we're getting full image URL from getProductImageUrl method
      */
     protected function _getMediaConfig()
     {
         return $this->_mediaConfig;
     }
+
+    /**
+     * Get product image URL from image filename and path
+     *
+     * @param string $image
+     * @return mixed
+     */
+    protected function getProductImageUrl($image)
```

Comment:
I would suggest making this private, as Magento discourages inheritance-based APIs.